### PR TITLE
adding jenkins agent version monitoring by updatecli

### PR DESCRIPTION
**Pull request by smerle33**
adding the jenkins agent in an arg to monitor it's version with updatecli, within the harness test too.
Closing #8

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -86,13 +86,17 @@ COPY --from=updatecli /usr/local/bin/updatecli /usr/local/bin/updatecli
 
 USER jenkins
 
-LABEL io.jenkins-infra.tools="golang,terraform,tfsec,packer,golangci-lint,aws-cli,yq,updatecli"
+## trick to have it working again
```
Comment (dduportal) [inline Dockerfile:89]:
```suggestion
## As per https://docs.docker.com/engine/reference/builder/#scope, ARG need to be repeated for each scope
```

**Comment by smerle33** [comment]
do not validate, still troubleshooting

**Comment by smerle33** [comment]
> do not validate, still troubleshooting

sounds good locally with comment on scmID ...

File: Dockerfile [hunk 2]
```diff
@@ -4,14 +4,18 @@
...
 
-FROM jenkins/inbound-agent:4.11-1-alpine-jdk11
-
+FROM jenkins/inbound-agent:${JENKINS_AGENT_VERSION}
 USER root
 
+COPY --from=gosource /usr/local/go/ /usr/local/go/
+# Configure Go
+ENV PATH /usr/local/go/bin/:$PATH
+
```
Comment (dduportal) [inline Dockerfile:18]: What is the reason of this change exactly?

**Comment by dduportal** [comment]
Merging even if the check fails:

- "Docker Build" step also fails on master branch
- updatecli still need the "scm" egg-and-chicken problem
